dallinger/db.py

```diff
@@ -67,9 +67,31 @@ def wrapper(*args, **kwargs):
 
 def init_db(drop_all=False):
     """Initialize the database, optionally dropping existing tables."""
-    if drop_all:
-        Base.metadata.drop_all(bind=engine)
-    Base.metadata.create_all(bind=engine)
+    try:
+        if drop_all:
+            Base.metadata.drop_all(bind=engine)
+        Base.metadata.create_all(bind=engine)
+    except OperationalError as err:
+        msg = 'password authentication failed for user "dallinger"'
+        if msg in err.message:
+            print
```

Comment:
I think this would be a bit clearer as a multiline string at module scope, like `dallinger.command_line.header`, but not a deal breaker.

Reply:
Also, these really should go to stderr, it will make it much clearer on Jupyter if nothing else.